base/ubi9/Dockerfile

```
@@ -152,6 +152,7 @@ RUN mv /usr/bin/podman "${ORIGINAL_PODMAN_PATH}"
 
 COPY --chown=0:0 entrypoint.sh /
 COPY --chown=0:0 .stow-local-ignore /home/tooling/
+COPY --chown=0:0 etc/.copy-files /home/tooling/
```

```suggestion
COPY --chown=0:0 .copy-files /home/tooling/
```
or need to move .copy-files into `developer-images/base/ubi9/etc` folder

---

Oops, thank you, fixed